pubspec.yaml

```diff
@@ -5,7 +5,7 @@ homepage: https://www.github.com/dart-lang/characters
 
 environment:
   # This must remain a tight constraint until nnbd is stable
-  sdk: '>=2.10.0-78 <2.10.0'
+  sdk: '>=2.10.0-137 <2.10.0'
```

Comment:
Note that this might be past the 2.10 stable branch point (and never included in 2.10)

Reply:
See https://github.com/dart-lang/sdk/issues/43485, also please mention there that you also were mislead by this

Reply:
I wasn't actually misled, I was just guessing wildly at why Travis gives errors in the sourcespan package, so I updated to the same SDK version as that package. Didn't work, will revert.